src/__tests__/operate.test.js

```diff
@@ -0,0 +1,16 @@
+import operate from '../logic/operate';
+
+describe('operate', () => {
+  test('should return 10 after adding 7+3', () => {
+    expect(operate('7', '3', '+')).toBe('10');
+  });
+  test('shoud return 4 after adding 7 - 3', () => {
+    expect(operate('7', '3', '-')).toEqual('4');
+  });
+  test('shoud return 21 after adding 7 x 3', () => {
+    expect(operate('7', '3', 'x')).toEqual('21');
+  });
+  test('shoud return 4 after adding 12 ÷ 3', () => {
+    expect(operate('12', '3', '÷')).toEqual('4');
+  });
+});
```

Comment:
- [ ] As required for the project all the functionalities & scenarios of the `operate.js` module to be covered using Jest, kindly make sure you add tests scenario to `%`, `raises an error when dividing number by 0`,....